.github/workflows/builds_and_tests.yml

```diff
@@ -134,6 +134,21 @@ jobs:
           name: libwasmvm
           path: ./api/libwasmvm.x86_64.so
 
+  build_static_lib:
+    runs-on: ubuntu-latest
+    steps:
+      - name: set up
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17.4
```

Comment:
We need to upgrade to 1.18 via another PR

Reply:
fixed